databricks/koalas/indexes.py

```diff
@@ -125,6 +125,34 @@ def spark_type(self):
         """ Returns the data type as defined by Spark, as a Spark DataType object."""
         return self.to_series().spark_type
 
+    @property
+    def has_duplicates(self) -> bool:
+        """
+        If index has duplicates, return True, otherwise False.
+
+        Examples
+        --------
+        >>> kdf = ks.DataFrame({'a': [1, 2, 3]}, index=list('aac'))
+        >>> kdf.index.has_duplicates
+        True
+
+        >>> kdf = ks.DataFrame({'a': [1, 2, 3]}, index=[list('abc'), list('def')])
+        >>> kdf.index.has_duplicates
+        False
+
+        >>> kdf = ks.DataFrame({'a': [1, 2, 3]}, index=[list('aac'), list('eef')])
+        >>> kdf.index.has_duplicates
+        True
+        """
+        scol = self._kdf._sdf.select(self._scol)
+        col = scol.columns[0]
+
+        count = scol.agg(F.count(col).alias('count')).collect()
+        dedup_count = scol.agg(F.countDistinct(col).alias('count')).collect()
```

Comment:
@charlesdong1991 can we do it in one pass?

```
df.select(F.count("id") == F.countDistinct("id")).first()
```

Reply:
Nice, this is much simpler! thanks! @HyukjinKwon